bin/hog/batchrecv.C

```diff
@@ -87,7 +87,7 @@ void read(gzbuffer* in, uint8_t* b, size_t n) {
   in->read(b, n);
 }
 
-#ifdef __clang__
+#if defined(__APPLE__) && defined(__MACH__)
```

Comment:
Can it be that an other commit message could explain the need (and consequences) to check two preprocessor symbols better?